.gitignore

```diff
@@ -106,5 +106,5 @@ ENV/
 *profraw
 
 # user defined (flo)
-protex/charmm-ff/out/
-protex/charmm-ff/traj/
+#protex/charmm-ff/out/
+#protex/charmm-ff/traj/
```

Comment:
this seems like two directories that we want to ignore?

Reply:
Yes, but these directories do not exist anymore, so we can delete it.